package.json

```diff
@@ -178,8 +178,8 @@
     "prettier": "^3.4.2",
     "pretty-quick": "^4.0.0",
     "process": "^0.11.10",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
```

Comment:
We should probably still test CI on react 18 though. It doesn't seem like `(props) => <div ref={props.ref}>` can work on react 18 without the `forwardRef`, but it works properly in 19.

19: https://codesandbox.io/p/sandbox/react-19-hws42c

18: https://codesandbox.io/p/sandbox/react-18-htw6s7

Reply:
Oh yeah, that is a great idea.
I was just not sure if we should do it in the same PR as this is already big...
But I think that given the circumstances, at least for some time, we should run both react 19 and 18 on CI. 🤔

Reply:
I did not test it out
but technically, this should allow you running additional workflows with R18
https://github.com/mui/mui-x/blob/master/test/README.md#ci

Better is off course to have it automated, but could be useful until that is set up